custom_visibility/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -194,7 +232,15 @@ public function elasticProcessDocumentPreIndex(Document $document, \SugarBean $b
     }
 
     /**
-     * {@inheritdoc}
+     * implementation of ElasticStrategyInterface interface
+     *
+     * Bean index fields to be indexed
+     *
+     * this demo function adds field, 'sales_stage', to be indexed
+     *
+     * @param string $module
+     * @param Visibility $provider
+     * @return array
```

Comment:
It would be nice to have a few words describing what the params and return array are